src/sentry/runner/commands/createuser.py

```diff
@@ -69,23 +69,27 @@ def createuser(email, password, superuser, no_password, no_input, force_update):
     from sentry.models import User
     from django.conf import settings
 
-    user = User(
+    fields = dict(
         email=email, username=email, is_superuser=superuser, is_staff=superuser, is_active=True
     )
 
-    if password:
-        user.set_password(password)
+    verb = None
+    try:
+        user = User.objects.get(username=email)
+    except User.DoesNotExist:
+        user = None
 
-    if User.objects.filter(username=email).exists():
+    if user is not None:
         if force_update:
-            user.save(force_update=force_update)
-            click.echo(f"User updated: {email}")
+            user.update(**fields)
+            verb = "updated"
         else:
             click.echo(f"User: {email} exists, use --force-update to force")
             sys.exit(3)
     else:
+        user = User(**fields)
```

Comment:
You could just use `User.objects.create` here as well

Reply:
Ah, didn't know about this one. I thought Python was the language where there was only one obvious way to do a certain thing? 🤣